components/segmenter/src/lib.rs

```diff
@@ -67,9 +67,10 @@
 //! Find all word boundaries:
 //!
 //!```rust
+//! use icu::locale::locale;
 //! use icu::segmenter::WordSegmenter;
 //!
-//! let segmenter = WordSegmenter::new_auto();
+//! let segmenter = WordSegmenter::try_new_auto(&locale!("en").into()).unwrap();
```

Comment:
Note: The request was for this to be

`new_auto()`

along with

`try_new_auto_with_options(WordSegmenterOptions::new_with_content_locale(&locale!("en")))`